doc/source/examples/JobAnalysis/job_analysis.ipynb.rst

```diff
@@ -258,17 +265,16 @@ Once an initial analysis has been created, the data can be further processed, de
 ======= ===== ======== =====================
 Name    OK    CPUTime  InputAmsNebIterations
 ======= ===== ======== =====================
-neb     False 0.523557 10
```

Comment:
Why are these values different?

Reply:
CPU time i.e. calculation time is expected to differ on re-running notebook